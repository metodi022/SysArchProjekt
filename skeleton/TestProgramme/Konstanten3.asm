ori $1, 0x4534
lui $1, 0x1423
lui $5, 0x4534
ori $5, 0x1423
sw $1, 4($0)
