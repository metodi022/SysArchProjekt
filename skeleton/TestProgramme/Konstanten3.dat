0x34214534
0x3c011423
0x3c054534
0x34a51423
0xac010004
