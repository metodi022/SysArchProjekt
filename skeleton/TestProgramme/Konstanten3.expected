@1 14230000
@5 45341423